packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -78,13 +79,19 @@ export const replayCommandHandler = async ({
   testRunId,
   replayEventsDependencies,
 }: ReplayOptions): Promise<ReplayResult> => {
+  const transaction = Sentry.startTransaction({
+    name: "replay.command_handler",
```

Comment:
For nicer Sentry logs:

```suggestion
    name: "replay.command_handler",
    description: "Handle the replay command",
    op: "replay.command_handler",
```